README.md

```diff
@@ -1,24 +1,7 @@
-# ForkDelta
-[![Chat on Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/MPvAfMa)
-[![Join us on Reddit](https://img.shields.io/badge/reddit-ForkDelta-red.svg)](https://www.reddit.com/r/ForkDelta/)
+# Agorex-io
+[Chat on Discord](https://discord.gg/GVuj6UY)
```

Comment:
Why not add the badge here too? Just use the same img link syntax as above: https://img.shields.io/badge/chat-on%20discord-7289da.svg

Reply:
Specifically change the exact syntax to:

[![Chat on Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/GVuj6UY)